src/main/java/beam/router/r5/OsmToMATSim.java

```diff
@@ -51,7 +50,6 @@
     public OsmToMATSim(final Network mNetwork, final CoordinateTransformation transformation,
```

Comment:
Can we just remove this from the constructor?

Reply:
yes